geometry_msgs/Point[] points
geometry_msgs/Point[] centers
float64 circle_number